pkg/scheduler/metrics/metrics.go

```diff
@@ -301,6 +308,38 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	ReadyTime = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "ready_time",
+			Help:      "Ready time for leader-scheduler to work.",
+		}, []string{"hostname"},
+	)
+
+	StartLatency = metrics.NewGaugeVec(
```

Comment:
Here I just want to get a single numerical value for start_latency rather than buckets. The numerical value can tell us how long we waited for the last time restart, and draw out a curve. 

Why not Histogram?  every restart can produce a data, but restarts are rare in normal circumstances, histogram is not so meaningful.

Reply:
I thought you wanted to get some analytics out of multiple restarts, e.g, average latency. If you just want a single numeric value, why not just log the latency? What does a metric really gives you?